File: sdkwork-app-sdk-csharp/Models/InvoiceItemVO.cs
using System;
using System.Collections.Generic;
using System.Text.Json.Serialization;

namespace App.Models
{
    public class InvoiceItemVO
    {
        public string? CreatedAt { get; set; }
        public string? UpdatedAt { get; set; }
        public string? ItemId { get; set; }
        public string? ProductName { get; set; }
        public string? Specification { get; set; }
        public string? Unit { get; set; }
        public double? Quantity { get; set; }
        public double? UnitPriceExcludingTax { get; set; }
        public double? AmountExcludingTax { get; set; }
        public double? TaxAmount { get; set; }
        public double? TotalAmount { get; set; }
        public double? TaxRate { get; set; }
    }
}

File: sdkwork-app-sdk-kotlin/src/main/kotlin/com/sdkwork/app/InvoiceItemVO.kt
package com.sdkwork.app

data class InvoiceItemVO(
    val createdAt: String? = null,
    val updatedAt: String? = null,
    val itemId: String? = null,
    val productName: String? = null,
    val specification: String? = null,
    val unit: String? = null,
    val quantity: Double? = null,
    val unitPriceExcludingTax: Double? = null,
    val amountExcludingTax: Double? = null,
    val taxAmount: Double? = null,
    val totalAmount: Double? = null,
    val taxRate: Double? = null
)
